modules/search/instant-search/components/search-results.jsx

```diff
@@ -9,17 +9,50 @@ import { h, Component } from 'preact';
 /**
  * Internal dependencies
  */
-import SearchResult from './search-result';
+import SearchResultMinimal from './search-result-minimal';
 
 class SearchResults extends Component {
+	render_result( result ) {
+		switch ( this.props.resultFormat ) {
```

Comment:
@bluefuton this property ^ which I think is correctly hooked up to the option passed from class.jetpack-search.php already